src/TicTacToe.elm

```diff
@@ -24,18 +25,25 @@ update : Msg -> Model -> Model
 update msg model =
   case msg of
     TakeTurn position ->
-      takeTurn position model
+      case model.playState of
+        InProgress model -> {playState = InProgress (takeTurn position model)}
+        NotStarted -> {playState = NotStarted}
```

Comment:
It can't be triggered, but elm won't let me not cover that branch in the case statement.